lib/platform/bitbucket/index.js

```diff
@@ -467,7 +476,11 @@ async function createPr(
     `/2.0/repositories/${config.repository}/pullrequests`,
     { body }
   )).body;
-  const pr = { id: prInfo.id, displayNumber: `Pull Request #${prInfo.id}` };
+  const pr = {
+    id: prInfo.id,
+    number: prInfo.id,
```

Comment:
Was this addressing an unrelated bug, or something else?

Reply:
This was addressing a bug that blocked the implementation of `addReviewers`. In [the caller of addReviewers](https://github.com/renovatebot/renovate/blob/9b9604ac1670fc31f30abdfdfbfe1b468b7f5e50/lib/workers/pr/index.js#L364) the pr.number is used. Without this fix, the first argument to addReviewers for Bitbucket was `undefined`

Reply:
Sounds like we should be returning only pr.number and not pr.id ! I'll look into this.